dotnet/src/Agents/UnitTests/OpenAI/Extensions/OpenAIServiceConfigurationExtensionsTests.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+using System;
+using Microsoft.SemanticKernel.Agents.OpenAI;
+using OpenAI.Files;
+using OpenAI.VectorStores;
+using Xunit;
+
+namespace SemanticKernel.Agents.UnitTests.OpenAI.Extensions;
+
+/// <summary>
+/// Unit testing of <see cref="OpenAIServiceConfigurationExtensions"/>.
+/// </summary>
+public class OpenAIServiceConfigurationExtensionsTests
+{
+    /// <summary>
+    /// Verify <see cref="OpenAIServiceConfiguration"/> can produce a <see cref="FileClient"/>
+    /// </summary>
+    [Fact]
+    public void OpenAIServiceConfigurationExtensionsCreateFileClientTest()
+    {
+        OpenAIServiceConfiguration configOpenAI = OpenAIServiceConfiguration.ForOpenAI("key", new Uri("https://localhost"));
+        Assert.IsType<FileClient>(configOpenAI.CreateFileClient());
+
+        OpenAIServiceConfiguration configAzure = OpenAIServiceConfiguration.ForAzureOpenAI("key", new Uri("https://localhost"));
+        Assert.IsType<FileClient>(configOpenAI.CreateFileClient());
```

Comment:
Arrange and Assert in different places when possible, for example here was clear to spot the error in the `configOpenAI` being used in `Azure` test.

Apply for others.

```suggestion
        // Arrange
        OpenAIServiceConfiguration configOpenAI = OpenAIServiceConfiguration.ForOpenAI("key", new Uri("https://localhost"));
        OpenAIServiceConfiguration configAzure = OpenAIServiceConfiguration.ForAzureOpenAI("key", new Uri("https://localhost"));
        
        Assert.IsType<FileClient>(configOpenAI.CreateFileClient());
        Assert.IsType<FileClient>(configAzure.CreateFileClient());
```

Reply:
Applied everywhere